src/Controller/ListPageRssController.php

```diff
@@ -164,12 +192,57 @@ public function build(NodeInterface $node): CacheableResponse {
    *
    * @param \Drupal\node\NodeInterface $node
    *   The node containing the list.
+   * @param \Drupal\Core\Cache\CacheableMetadata $cache_metadata
+   *   The current cache metadata.
    *
    * @return array
    *   Array of items to be rendered.
    */
-  protected function getItemList(NodeInterface $node): array {
-    return [];
+  protected function getItemList(NodeInterface $node, CacheableMetadata $cache_metadata): array {
+    $configuration = ListPageConfiguration::fromEntity($node);
+    $execution_result = $this->listExecutionManager->executeList($configuration);
+    $query = $execution_result->getQuery();
+    $results = $execution_result->getResults();
+    $cache_metadata->addCacheableDependency($query);
+    $cache_metadata->addCacheableDependency($results);
+    $cache_metadata->addCacheTags(['search_api_list:' . $query->getIndex()->id()]);
+    $result_items = [];
+    foreach ($results->getResultItems() as $item) {
+      /** @var \Drupal\Core\Entity\ContentEntityInterface $entity */
+      $entity = $item->getOriginalObject()->getEntity();
+      $cache_metadata->addCacheableDependency($entity);
+      $creation_date = $entity->get('changed')->value;
+      if ($entity->hasField('body')) {
+        $description = $entity->body->view([
+          'type' => 'text_default',
+          'label' => 'hidden',
+          'settings' => [],
+        ]);
+        $description = Html::escape($this->renderer->renderPlain($description[0]));
+      }
+      $description = $description ?? '';
+      $result_item = [
+        '#theme' => 'oe_list_pages_rss_item',
+        '#title' => $entity->label(),
+        '#link' => $entity->toUrl('canonical', ['absolute' => TRUE]),
+        '#guid' => $entity->toUrl('canonical', ['absolute' => TRUE]),
+        '#description' => $description,
+        '#item_elements' => [
```

Comment:
You didn't key this array. So when overriding we cannot easily access it in the array in case we wanna change pubDate. See my other comment!